custom/abt/reports/fixture_utils.py

```diff
@@ -230,3 +239,27 @@ def dict_values_in(
         if dict_[key] not in values:
             return False
     return True
+
+
+def dict_value_not(
```

Comment:
Looks pretty simple, so may be a test case is not required?

Reply:
You're probably right.

I always worry that my description in a docstring isn't clear, and an example helps. ... I think I'm going to leave it for now.